mlir/utils/performance/perfRunner.py

```diff
@@ -27,6 +27,8 @@
 ROCPROF = '/opt/rocm/bin/rocprof'
 MIOPENDRIVER = '/opt/rocm/bin/MIOpenDriver'
 BENCHMARKING_RESULT_FILE_NAME = 'results.stats.csv'
+BENCHMARKING_METRICS_FILE_NAME = 'results.csv'
+ROCMLIR_INPUT_METRICS_FILE_NAME = 'results.txt'
```

Comment:
I would call this something like `rocmlir_metrics.txt`. This is the file containing the metrics, not the results

Reply:
Yes I have problem if change the name like (rocmlir_metrics.txt) in output file I don't have number of TFlops got NaN, because this line when call [ROCPROF, '-i', file_path, '--stats', paths.mlir_paths.cpu_runner_path] + mlir_cpu_runner_args will create rocmlir_metrics.stats.csv output file not results.stats.csv and that is a problem.

Reply:
With '-i' and '--stats' together will create a output file from input name. For this reason, I tried to rename everything to rocmlir_metrics, but then there is a problem with the tuning runner because it only calls '--stats' and by default results.csv text output files are created.

Reply:
Well you can specify the `-o` option with `rocprof`. I just tested this:
```
rocprof --stats  -i prof.txt -o mlir-prof.csv 
```
And I get a `mlir-prof.stats.csv` and a `mlir-prof.csv`. So I guess you need to do :
```
[ROCPROF, '-i', ROCMLIR_INPUT_METRICS_FILE_NAME , '--stats', paths.mlir_paths.cpu_runner_path, -o, BENCHMARKING_METRICS_FILE_NAME ] 
```
And I would stick with:
```
BENCHMARKING_RESULT_FILE_NAME = 'results.stats.csv'
BENCHMARKING_METRICS_FILE_NAME = 'results.csv'
ROCMLIR_INPUT_METRICS_FILE_NAME = 'rocmlir_metrics.txt'
```

Reply:
Thank you @giuseros, I will try to do that.